src/app/(BottomTabNavigation)/AllCases/Forms/styles.ts

```diff
@@ -0,0 +1,58 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../../../styles/colors';
+
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    backgroundColor: colors.white,
+  },
+  contentContainer: {
+    width: '82%',
+    height: '84%',
+  },
+  linkContainer: {
+    height: '16%',
```

Comment:
Let's use pixels here (and below) if possible. I think we should only use percentages for larger containers or places where we use 100%.

Reply:
I agree. One thing I like about percentages, in this case, is the scroll view container will stretch until it reaches the fixed link container at the bottom. When I set exact pixel heights, I can't replicate this. Do you know a way around this?

Reply:
Hmm no I can't seem to figure it out either. Maybe it makes sense to use % in this case then.